test/functional/domains_controller_test.rb

```diff
@@ -1,15 +1,11 @@
 require 'test_helper'
+require 'functional/shared/basic_rest_response_test'
```

Comment:
Yeah, that's OK by me I think.  Since it's always going to be required by running any of these test files I think not having to require it in each file will save some lines.